lib/utils/prop-types.js

```diff
@@ -6,30 +6,30 @@
  */
 
 module.exports = new Set([
+	"amd",
+	"bail",
+	"cache",
 	"context",
 	"devServer",
 	"devtool",
 	"entry",
 	"externals",
+	"merge",
+	"mode",
 	"module",
 	"node",
 	"output",
+	"parallelism",
 	"performance",
 	"plugins",
-	"resolve",
-	"target",
-	"watch",
-	"watchOptions",
-	"stats",
-	"mode",
-	"amd",
-	"bail",
-	"cache",
 	"profile",
-	"merge",
-	"parallelism",
 	"recordsInputPath",
 	"recordsOutputPath",
 	"recordsPath",
-	"resolveLoader"
+	"resolve",
```

Comment:
Bad rebase?

Reply:
It includes all properties. I think Git/GitHub diff might not interpret it as sorting but only show changes in code based on line number. That's why we see such confusing diffs. You can verify it from the [current file](https://github.com/webpack/webpack-cli/blob/master/lib/utils/prop-types.js).